locales-pending/emails-all.ftl

```diff
@@ -0,0 +1,69 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Strings in this file are not yet final, and thus should not be localised yet.
+
+email-header-logo-alt = { -brand-mozilla-monitor }
+email-header-button-sign-in = Sign in
+
+email-footer-support-heading = Questions about { -brand-mozilla-monitor }?
+email-footer-support-content = Visit our <support-link>Support Center</support-link> for help
+# TODO: Check with flod - do we need a token for "Mozilla Support"?
+email-footer-reason-subscriber = You’re receiving this automated email as a subscriber of { -brand-mozilla-monitor }. If you received it in error, no action is required. For more information, please visit <support-link>{ -brand-Mozilla } Support</support-link>.
```

Comment:
```suggestion
email-footer-reason-subscriber = You’re receiving this automated email as a subscriber of { -brand-mozilla-monitor }. If you received it in error, no action is required. For more information, please visit <support-link>{ -brand-mozilla } Support</support-link>.
```

We should probably remove the brand definition `-brand-Mozilla` to avoid confusion, and fix `parent-company` to use `-brand-mozilla`.

Reply:
Is `parent-company` one of the deleted strings? I can't find it in `locales/en` or outside of `locales`.

I've consolidated the brand names in one place and removed a couple of unused ones in 961cfbdd3, but would appreciate another look in case I messed anything up.

Reply:
Oh, that's a bug… The script doesn't remove files that have been removed from this repository (it's the case for `landing.ftl`). I'll need to fix that.